containers/jenkins-master/config/jobs/github-snappy-daily-ppa/config.xml

```diff
@@ -71,7 +71,8 @@ set -ex
 export GOPATH=$WORKSPACE
 cd $GOPATH/src/github.com/ubuntu-core/snappy
 
-sudo mk-build-deps -i -t "apt-get --yes --no-install-recommends"
+sudo apt update
+sudo apt build-dep -y ./
```

Comment:
That should be taken care-of in https://github.com/ubuntu-core/snappy-jenkins/blob/master/containers/jenkins-slave-xenial/Dockerfile#L21
Maybe I forgot to install it manually in one of the already deployed slaves, but in the next deploy this will make sure that it's preinstalled in all of them.